numpy/linalg/umath_linalg.c.src

```diff
@@ -706,6 +706,11 @@ dump_@TYPE@_matrix(const char* name,
  *****************************************************************************
  */
 
+#ifndef min
+#define min(x,y) ((x<y)?x:y)
```

Comment:
Thought that I could get a pass here by virtue of copying `/numpy/random/mtrand/distributions.c
`. Should I fix the macros there too?